src/core/utils.ts

```diff
@@ -83,25 +80,19 @@ export const uriToHttp = (uri: string): string[] => {
   }
 };
 
-export const getAddressFromEnsName = (
-  ensName: string,
-  library: JsonRpcProvider
-) => {
+export const getAddressFromEnsName = (ensName: string, library: any) => {
   return library.resolveName(ensName);
 };
 
-export const getEnsNameFromAddress = (
-  address: string,
-  library: JsonRpcProvider
-) => {
-  if (isAddress(address)) {
+export const getEnsNameFromAddress = (address: string, library: any) => {
+  if (utils.isAddress(address)) {
     return library.lookupAddress(address);
   } else return null;
 };
 
 // a type guard for MM specific methods/properties
 export const isMetaMaskProvider = (
-  library: MetaMaskProvider | JsonRpcProvider
+  library: any
```

Comment:
`any` typeGuard 😅

Reply:
For some reason the JsonRpcProvider interfaces from ethers and usedapp are incompatible not sure why